Facebook Conversions API, Google Enhanced Conversions, and Google Analytics 4 destinations (#2278)
segmentio/segment-docs: src/connections/destinations/catalog/google-analytics/index.md

#### Which Google destination should I use?

> If your Google Measurement ID starts with a G, you are using G-Codes from Google Analytics 4, and should consider using [Segment's Google Analytics 4 destination](/docs/connections/destinations/catalog/actions-google-analytics-4/).

Reviewer: ```suggestion
If your Google Measurement ID starts with a G, you're using G-Codes from Google Analytics 4, and should consider using [Segment's Google Analytics 4 destination](/docs/connections/destinations/catalog/actions-google-analytics-4/).
```

Although GA4 is now the default when you create a new property, you can still [create a Universal Analytics property](https://support.google.com/analytics/answer/10269537). You can use a UA property with [Segment's Google Universal Analytics destination](/docs/connections/destinations/catalog/google-analytics/).

Different Measurement IDs begin with different prefixes, which indicate which Google destination you should use.

| Prefix | Google Account type       | Segment Settings    |
| ------ | -------------------------- | ----------------- |

> | UA     | Your global site tag is controlled by Google Universal Analytics. The ID is your Google Universal Analytics Measurement ID. To find the property associated with this ID, use the [account search feature](https://support.google.com/analytics/answer/6100731) in Google Universal Analytics. If the property does not appear, you probably do not have access to it. | [Google Universal Analytics](/docs/connections/destinations/catalog/google-analytics/): Tracking ID           |

Reviewer: ```suggestion
| UA     | Your global site tag is controlled by Google Universal Analytics. The ID is your Google Universal Analytics Measurement ID. To find the property associated with this ID, use the [account search feature](https://support.google.com/analytics/answer/6100731) in Google Universal Analytics. If the property doesn't appear, you probably don't have access to it. | [Google Universal Analytics](/docs/connections/destinations/catalog/google-analytics/): Tracking ID           |
```
